### ISSUE TITLE
Neural Process

### ISSUE THREAD

**Issue by JisuHann** [JisuHann/One-day-One-paper#22]
# Neural Process
1. Conditional Neural Processes(ICML2018) [paper](https://arxiv.org/pdf/1807.01613.pdf)
2. Neural Processes(ICML2018) [paper](https://arxiv.org/pdf/1807.01622.pdf)
3. Convolutional Conditional Neural Processes(ICLR2020) [paper](https://arxiv.org/pdf/1910.13556.pdf)

# CNP: Conditional Neural Processes(ICML2018)
![](https://ameroyer.github.io/reading-notes/images/posts/conditional_neural_processes.png)
- define conditional distributions over functions given a set of observations
1. a family of neural models (that represents solutions to the supervised problem) -> embedding h 
    - trained by sampling a random dataset
2. end-to-end training approach to learning them, that aggregate NN (using GP) -> aggregator r
3. conditions the function g 

- Advantage
    - make accurate predictions after observing only a handful of training data points -> scale to complex functions and large datasets
- On classification task
    - parametrizes the logits of the class probabilities p_c over the c classes of a categorical distributions  
    - Apply on Omniglot dataset for one0shot classification

# LNP: (Latent) Neural Processes(ICML2018)
- a class of neural latent variable models

### GP, DNN, Meta-learning
- Gaussian Process
    - design appropriate priors
    - on CNP, use flexibility of stochastic processes 
          - ability to extract prior knowledge directly from training data tied with scalability at test time
          - models that directly parameterize conditional stochastic processes
    - on CNP, not implement Bayesian inference directly and not true that the conditional distributions will be consistent with respect to some prior process
- Deep Neural Networks
    - on CNP, structured as DNN

**Comment by JisuHann** [comment]
# ConvCNP: Convolutional Conditional Neural Process (ICLR2020)
- NP family that models Translation Equivariance functions on sets
- Model embeds data sets into an infinite-dimensional function space as opposed to a finite-dimensional vector 

# ConvLNP: Meta-Learning Stationary Stochastic Process Prediction with Convolutional Neural Processes

**Comment by JisuHann** [comment]
# ConvLNP: Meta-Learning Stationary Stochastic Process Prediction with Convolutional Neural Processes

**Comment by JisuHann** [comment]
# SNP: Sequential Neural Processes

**Comment by JisuHann** [comment]
# ASNP: Attentive Sequential Neural Processes

**Comment by JisuHann** [comment]
# RMR: Robustifying Sequential Neural Processes
